lib/ui/page/home/page/chat/info/view.dart

```diff
@@ -373,7 +373,10 @@ class ChatInfoView extends StatelessWidget {
                   offset: const Offset(0, -1),
                   child: Transform.scale(
                     scale: 1.15,
-                    child: SvgLoader.asset('assets/icons/copy.svg', height: 15),
+                    child: SvgImage.asset(
+                      'assets/icons/copy.svg',
+                      height: 15,
```

Comment:
Будет ли форматирование лучше?

```suggestion
                      height: 15
```

Reply:
@SleepySquash Тут тоже сложно сказать, но все же лучше убрать думаю.